gobuffalo/azuredeploy.parameters.json

```diff
@@ -0,0 +1,5 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {}

```

Comment:
add a parameter here, I think that's the problem with CI

eg

"name": {"value": "GEN-UNIQUE"}

Reply:
Fixed locally